pkg/logging/logrus_logr.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package logging
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
```

Comment:
Oh right, another reason logr is hard to work with is because writing a shim is non trivial. totally fine here, just a followup from our chat yesterday